src/controllers/subscription.js

```diff
@@ -96,7 +96,10 @@ const updateSub = (req, res) => {
           message: `${id} Subscription was not found`,
         });
       }
-      return res.status(200).json(result);
+      return res.status(200).json({
+        message: 'Subscription Updated',
+        data: result,
```

Comment:
add error in response

```suggestion
        data: result,
        error: false,
```